erikiado/jp2_online
# Task/forma validacion login

#### What's this PR do?

#### Where should the reviewer start?
It creates the views, and methods necessary to login a user.

#### How should this be manually tested?
The tester should create a user, and try to login at localhost:8000/login, then test invalid credentials, and then test submitting the form with empty fields.

#### Any background context you want to provide?
N/A


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
